oauth.go

```diff
@@ -148,3 +150,30 @@ func (app App) VerifyWebhookRequestVerbose(httpRequest *http.Request) (bool, err
 
 	return HMACSame, nil
 }
+
+func (app App) VerifySignature(u *url.URL) bool {
```

Comment:
Please could you add a comment to this method to explain its purpose

Reply:
@ysurtayev-bold how's this? (added commit)